ios/BT/ExposureManager.swift

```diff
@@ -525,4 +523,61 @@ private extension ExposureManager {
     localUncompressedURLs = []
     downloadedPackages = []
   }
+
+  func processExposureDetectionSummary(_ summary: ENExposureDetectionSummary?, completion: @escaping (([ExposureDetectionSummary]) -> Void)) {
+    if let summary = summary {
+
+      // Serialize exposure detection summary
+      let exposureDetectionSummary = ExposureDetectionSummary(summary)
+
+      // Prune exposure detection summaries >14 days from realm
+      BTSecureStorage.shared.pruneExposureDetectionSummaries()
+
+      // Fetch exposure summaries <14 days
+      let unusedExposureSummaries = BTSecureStorage.shared.allExposureDetectionSummaries()
+
+      // Store new exposure summary in realm
+      BTSecureStorage.shared.storeExposureDetectionSummary(exposureDetectionSummary)
+
+      // Call scoring server
+      apiClient.request(ExposureDetectionSummaryListRequest.post(exposureDetectionSummary,
+                                                                 unusedExposureSummaries),
+                        requestType: .scoring) { result in
+                          switch result {
+                          case .success(let scoringServerResponse):
+                            let summaries = scoringServerResponse.notifications.map { $0.exposureSummaries }.flatMap { $0 }
+                            completion(summaries)
+                          case .failure:
+                            completion([])
+                          }
+      }
+    }
+  }
+
+  func processScoringServerResponse(_ summaries: [ExposureDetectionSummary],
+                                    processedFileCount: Int,
+                                    lastProcessedUrlPath: String,
+                                    progress: Progress,
+                                    completionHandler: @escaping ((ExposureResult) -> Void)) {
+
+    let newExposures = (summaries).map { summary in
+      Exposure(id: UUID().uuidString,
+               date: summary.startOfDateReceived)
+    }
+
+    // Remove "used" exposure summaries from realm
+    BTSecureStorage.shared.pruneExposureDetectionSummaries(summaries)
+
+    // Store new exposures in realm
+    btSecureStorage.storeExposures(newExposures)
```

Comment:
By "exposures" here, you mean the exposures that led to a notification?
If so, I'd consider different naming here - as a reader, it's not obvious what the difference is here.
And consider storing the whole notification object instead?